static/app/views/settings/projectDebugFiles/index.tsx

```diff
@@ -137,6 +148,32 @@ class ProjectDebugSymbols extends AsyncView<Props, State> {
     });
   }
 
+  getFieldsSymbolSources() {
+    const {organization} = this.props;
+
+    if (!organization.features?.includes('app-store-connect')) {
+      return fields.symbolSources;
+    }
+
+    const fieldsSymbolSources = fields.symbolSources as any;
+
+    if (
+      !!fieldsSymbolSources.addDropdown.items.find(
+        item => item.value === 'appStoreConnect'
+      )
+    ) {
+      return fields.symbolSources;
+    }
+
+    fieldsSymbolSources.addDropdown.items.push({
+      value: 'appStoreConnect',
```

Comment:
before removing the feature flag, this object will be moved to the https://github.com/getsentry/sentry/blob/2bb8a0ca24c66ca4ad336bce025babe1269cc189/static/app/data/forms/projectDebugFiles.tsx#L97